crates/client/src/account/register.rs

```diff
@@ -1,24 +1,32 @@
 use crate::helpers::get_by_id::get_by_id;
 use crate::route::{Route, Router};
+use reqwest::Client;
+use serde::Serialize;
+use sha2::{Digest, Sha256};
+use wasm_bindgen_futures::spawn_local;
 use web_sys::console;
 use yew::{function_component, html, Callback, Html};
 
 #[function_component(Register)]
 pub fn register() -> Html {
     let onsubmit = Callback::from(move |e: yew::events::SubmitEvent| {
         e.prevent_default();
+        spawn_local(async move {
+            let email_input = get_by_id("email");
+            let username_input = get_by_id("username");
+            let password_input = get_by_id("password");
+            let hashed_password = hash_password(&password_input);
 
-        let email_input = get_by_id("email");
-        let username_input = get_by_id("username");
-        let password_input = get_by_id("password");
-
-        console::log_1(
-            &format!(
-                "Email: {}, Password: {}, Username: {}",
-                email_input, password_input, username_input
-            )
-            .into(),
-        );
+            let _ = register_in_backend(&hashed_password, &username_input, &email_input).await;
```

Comment:
When function will return error then this error will be ignored, you don't proceed it in any way

Reply:
I fixed